google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -588,7 +635,8 @@ public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext
         BulkMutation.create(tableId),
         settings.bulkMutateRowsSettings().getBatchingSettings(),
         clientContext.getExecutor(),
-        bulkMutationFlowController);
+        bulkMutationFlowController,
+        ctx == null ? clientContext.getDefaultCallContext() : ctx);
```

Comment:
isnt this done above?

Reply:
Slightly cleaner: Objects.firstNonNull(ctx, clientContext.getDefaultCallContext())